learning/features/assetlib/using_assetlib.rst

```diff
@@ -0,0 +1,185 @@
+.. _doc_using_assetlib:
+
+Using the AssetLib
+==================
+
+On the website
+--------------
+
+Overview
+~~~~~~~~
+
+As mentioned before, you can access the web frontend of the AssetLib
+on `Godot's official website <https://godotengine.org/asset-library>`_, and this
+is what it looks like when you first visit it:
+
+|image0|
+
+At the top you see the **header** which takes you to various other parts of the
+AssetLib - at the moment it's empty as we are not logged in.
+
+Searching
+~~~~~~~~~
+
+In the center is the **search bar + settings** section, and the **assets** section
+below it - this lets you filter out certain kinds of assets based on a host
+of criteria. These include the asset **category** (such as 2D tools, scripts
+and demos), **engine version** they are intended for, **sorting order** (by
+update date, by name, etc.) and **support level**.
+
+While most other filter settings should be fairly self-explanatory, it's worth
+going over what "support level" means in the Asset Library.
+Currently there are three support levels, and each asset can belong to only one.
+
+**Official** assets are created and maintained by the official Godot Engine
+developers. Currently, these include the official engine demos, which showcase
+how various areas of the engine work.
+
+**Community** assets are submitted and maintained by the members of the
+Godot community.
+
+**Testing** assets are works-in-progress, and may contain bugs and usability
+issues. They are not recommended for use in serious projects, but you are
+encouraged to download, test them, and submit issues to the original authors.
+
+You can mix and match any of the search filters and criteria, and upon clicking
+the Search button, receive the list of all assets in the Library that match them.
+
+|image1|
+
+Note that the search results are not updated in real-time, so you will have to
+re-submit the search query each time you change the query settings.
+
+Breakdown of an asset
+~~~~~~~~~~~~~~~~~~~~~
+
+Now let's take a look at what an asset's page looks like and what it contains.
+
+|image2|
+
+1. Asset's thumbnail/icon.
+2. Asset's name.
+3. Current version number of the asset.
+4. Asset's average rating, displayed in stars. (This is currently unimplemented.)
```

Comment:
Is "unimplemented" a word? If it were, I think it would imply something that was removed. "not implemented" might read better.